ReSharper.FSharp/test/data/features/quickFixes/removeXmlDoc/XmlDoc 02.fs

```diff
@@ -1,2 +1,2 @@
-() /// Uwu{caret}
+()/// Uwu{caret}
```

Comment:
Why remove the space in the test data? The test should show that quick fix removes it properly, shouldn't it?

Reply:
I wanted to separately check the edge case when there is nothing to additionally delete between the comment and the node to the left. The `XmlDoc 03 - Remove line` test checks to remove spaces and newline to the left of the comment. 
Perhaps, we should separately check the removal of spaces between the comment and the node to the left, but it seemed redundant to me.